generator/osm2type.cpp

```diff
@@ -325,7 +325,7 @@ void LeaveLongestTypes(vector<generator::TypeStrings> & matchedTypes)
 
 void MatchTypes(OsmElement * p, FeatureBuilderParams & params, function<bool(uint32_t)> const & filterType)
 {
-  auto static const rules = generator::ParseMapCSS(GetPlatform().GetReader(MAPCSS_MAPPING_FILE));
+  auto static const rules = generator::ParseMapCSS(GetPlatform().GetReader("mapcss-mapping.csv"));
```

Comment:
What went wrong here?

Reply:
Seems I based this off an older base version and not the head of the file which also explains the comment change below, too